tests/snapshot/snapshot.go

```diff
@@ -251,9 +270,23 @@ func createPostBackupWorkloads(t *testing.T, client *rancher.Client, clusterID s
 	steveclient, err := client.Steve.ProxyDownstream(clusterID)
 	require.NoError(t, err)
 
-	postDeploymentResp, err := deploy.CreateDeployment(steveclient, workloadNamePostBackup, postBackupDeployment)
+	postDeploymentResp, err := steveclient.SteveType(DeploymentSteveType).Create(postBackupDeployment)
 	require.NoError(t, err)
 
+	err = kwait.PollUntilContextTimeout(context.TODO(), timeouts.FiveSecondTimeout, timeouts.FiveMinuteTimeout, true, func(ctx context.Context) (done bool, err error) {
```

Comment:
nit - not a big deal if you don't want to do this. Are we able to put this in a helper function in this same file for abstraction purposes? Also, I don't remember if `active` is in the `defaults` folder. If not, can you just add a const in this file?